src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Lottery();
+        Lottery ball1 = new Lottery();
+        Lottery ball2 = new Lottery();
+        System.out.println(ball.getRandomBall());
+        System.out.println(ball1.getRandomBall());
+        System.out.println(ball2.getRandomBall());
```

Comment:
[Use for loop for creating several objects of the same class](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class)
Also, pay attention  that one object  of the lottery enough